relational_operators/CreateIndexOperator.hpp

```diff
@@ -75,7 +83,8 @@ class CreateIndexOperator : public RelationalOperator {
 
  private:
   CatalogRelation *relation_;
-  const std::string &index_name_;
+  std::string index_name_;
```

Comment:
@zuyu Yes, it shouldn't change during query optimization as the index_name string is part of the phyiscal plan that is generated during the query optimization. But the operator is part of the DAG that gets passed to Foreman. I am may wrong here, but is the plan kept around till the DAG is actually executed? If it is, then the references will never changes.